### PR TITLE
COVID-19 alerts

### PR DESCRIPTION
**Pull request by JakeSidSmith**
Screenshot 2020-03-13 at 12 09 00
Screenshot 2020-03-13 at 12 09 14
Screenshot 2020-03-13 at 12 09 23

### CODE REVIEW

**Comment by dvberzon** [comment]
Any way of not having the "sorry all events are cancelled" on the past events?

**Comment by JakeSidSmith** [comment]
But what if people accidentally go to a past event and start the outbreak years ago? 😮

**Comment by dvberzon** [comment]
If all past events are cancelled, then we never met!

File: _includes/partials/covidAlert.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<p class="alert">
+  {{{ iconmonstr 'alert' }}} Sorry, all events are cancelled until further notice.
```
Comment (larister) [inline _includes/partials/covidAlert.html]:
```suggestion
  {{{ iconmonstr 'alert' }}} Sorry, all events are cancelled until further notice due to [COVID-19](https://www.gov.uk/government/topical-events/coronavirus-covid-19-uk-government-response).
```

File: _includes/partials/covidBanner.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<div class="alert-banner">
+  <h3>
+    Sorry, all events are cancelled until further notice.
+  </h3>
+  <p>
+    We're taking precautions due to the increasing risk of COVID-19 (Coronavirus), and unfortunately will not be running any events until further notice.
```
Comment (larister) [inline _includes/partials/covidBanner.html]:
```suggestion
    We're taking precautions due to the increasing risk of [COVID-19 (Coronavirus)](https://www.gov.uk/government/topical-events/coronavirus-covid-19-uk-government-response), and unfortunately will not be running any events until further notice.
```

**Comment by larister** [comment]
Thanks so much for doing this @JakeSidSmith 🙏

**Comment by JakeSidSmith** [comment]
Screenshot 2020-03-13 at 12 30 09

**Pull request merged**